bin/seed_test_users.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+numberExpression='^[0-9]+$'
+databasePath=var/user-lifecycle.sqlite
+
+if ! [[ $1 =~ $numberExpression ]] ; then
+   echo "Usage: The first argument must be a number. Indicating the number of identities you'd like to seed to the last_login table" >&2; exit 1
+fi
+
+if ! [[ $2 =~ $numberExpression ]] ; then
+   echo "Usage: The second argument must be a number. Indicating the number of days since last login for all users" >&2; exit 1
+fi
+
+if ! [ -x "$(command -v sqlite3)" ]; then
+    echo "The SQLite binary is not available"
```

Comment:
You might want to exit here if the binary is not available

Reply:
Yes, completely missed that :ok_hand: